README.md

```diff
@@ -1,31 +1,71 @@
 # Python Bindings for UCX
 
-## Installing preliminary Conda packages.
+# Detailed Build and instructions
 
-Some preliminary Conda packages can be installed as so. Replace `<CUDA
-version>` with either `9.2` or `10.0`.
-
-```
-conda create -n ucx -c conda-forge -c jakirkham/label/ucx cudatoolkit=<CUDA version> ucx-proc=*=gpu ucx ucx-py python=3.7
-```
```

Comment:
Looks like this got dropped accidentally. Am adding it back in PR ( https://github.com/rapidsai/ucx-py/pull/181 ). Please take a look. 🙂